Source/Core/AttributeCompression.js

```diff
@@ -303,8 +304,9 @@ define([
         //>>includeEnd('debug');
 
         var temp = compressed / 4096.0;
-        result.x = Math.floor(temp) / 4096.0;
-        result.y = temp - Math.floor(temp);
+        var xZeroTo4095 = Math.floor(temp);
```

Comment:
Yeah that's a good point, we could do this whole thing as integers and only make it a float at the end.